.github/workflows/rust.yml

```diff
@@ -64,11 +64,11 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Install GTK4 (Ubuntu)
-        run: sudo apt-get update && sudo apt-get install --no-install-recommends -y libgtk-4-dev
+        run: sudo apt-get update && sudo apt-get install --no-install-recommends -y libgtk-4-dev libatk1.0-dev
```

Comment:
This is not mentioned in documentation of `tray-icon`, which likely means it should not be added and instead we should include the dependencies actually mentioned there that will likely bring `libatk1` as an indirect dependency. If this is sufficient then fine, but it does look odd.